src/main/java/ghbot/executor/FindExecutor.java

```diff
@@ -0,0 +1,48 @@
+package ghbot.executor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * FindExecutor Class.
+ * Executes "Find" command.
+ */
+public class FindExecutor extends Executor {
+    private String description;
+    private String executeStr;
+
+    /**
+     * FindExecutor Constructor.
+     * @param description A description of the word the user is finding.
+     */
+    public FindExecutor(String description) {
+        this.description = description;
+        this.executeStr = "";
+    }
+
+    /**
+     * Prints all the tasks that matches the description.
+     */
+    @Override
+    public String execute() {
+        List<String> matchedLst = new ArrayList<>();
+        for (int i = 0; i < this.taskList.taskSize(); i++) {
+            String[] words = this.taskList.getTask(i).toString().split(" ");
+            for (int j = 0; j < words.length; j++) {
+                if (this.description.equalsIgnoreCase(words[j])) {
```

Comment:
I like that you used this function equalsIgnoreCase to ensure that the code works even with different case words. I did not know about this function so I learnt about it 👍